javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/config/ConfigCustomizer.java

```diff
@@ -16,7 +16,10 @@
  *
  * <p>This is a service provider interface that requires implementations to be registered in a
  * provider-configuration file stored in the {@code META-INF/services} resource directory.
+ *
+ * @deprecated Use the {@link ConfigPropertySource} SPI instead.
```

Comment:
just confirming there's no replacement for `the customize()` method in this PR?

Reply:
No, not in this PR